packages/domain-picker/src/domain-picker/style.scss

```diff
@@ -99,61 +107,13 @@ $domain-picker__suggestion-item-height: 24px;
 			opacity: 1;
 		}
 	}
-
-	&.components-button {
-		&.is-tertiary {
-			color: var( --color-text );
-
-			&:not( :disabled ):not( [aria-disabled='true'] ):hover {
-				background: var( --color-surface-backdrop );
-				color: var( --color-text );
-			}
-		}
-	}
-
-	input[type='radio'].domain-picker__suggestion-radio-button {
-		width: 16px;
-		height: 16px;
-		margin: 3px 12px 0 0; // 3px cosmetic alignment
-		min-width: 16px; // prevents long domain from squishing the radio button
-		padding: 0;
-		vertical-align: middle;
-		position: relative;
-
-		&:checked {
-			border-color: var( --studio-blue-30 );
-			background-color: var( --studio-blue-30 );
-
-			&::before {
-				content: '';
-				width: 12px;
-				height: 12px;
-				border: 2px solid white;
-				border-radius: 50%;
-				position: absolute;
-				margin: 0;
-				background: transparent;
-			}
-
-			&:focus {
-				border-color: var( --studio-blue-30 );
-				box-shadow: 0 0 0 1px var( --studio-blue-30 );
-			}
-		}
-
-		&:not( :disabled ):focus,
-		&:not( :disabled ):hover {
-			border-color: var( --studio-blue-30 );
-			box-shadow: 0 0 0 1px var( --studio-blue-30 );
-		}
-	}
 }
 
 .domain-picker__suggestion-item-name {
 	flex-grow: 1;
 	margin-right: 24px;
 	letter-spacing: 0.4px;
-	min-height: $domain-picker__suggestion-item-height;
+	// min-height: $domain-picker__suggestion-item-height;
```

Comment:
Let's remove the line altogether?

Reply:
Ah thanks! Must've slipped in! That's what extra pair of eyes is for! :)